.golangci.yaml

```diff
@@ -37,12 +41,23 @@ linters:
     - unconvert
     - unused
     - varcheck
+    - wastedassign
     - whitespace
 
+output:
+  uniq-by-line: false
+
+issues:
+  max-issues-per-linter: 0
+  max-same-issues: 0
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - funlen
+        - bodyclose
+
 linters-settings:
-  gocritic:
-    disabled-checks:
-      # TODO: re-enable after figuring out why runcommand fails on go-critic..
-      - unlambda
   goconst:
     min-len: 5
+  predeclared:
+    ignore: "new"
```

Comment:
I added this ignore to Terraform provider as it had many instances of `old, new := d.GetChange(...)`, probably would have been good to add explanation there as well 😅 If `upctl` does not have `new` declarations at the moment this could be not ignored.